### DISCUSSION TITLE
How to use dynamic routes?

### DISCUSSION THREAD

**Discussion by AmirHo3einJS** [thunderclient/thunder-client-support#616] [Q&A] [answered]
Hello, I wonder whether there is a way to use dynamic routes like `http://localhost:3000/test/:id` like the postman.

**Comment by rangav** [comment]
@AmirHo3einJS you can use path variables for this

See docs here
https://github.com/rangav/thunder-client-support#path

**Comment by rangav** [comment, marked as answer]
@AmirHo3einJS you can use path variables for this

See docs here
https://github.com/rangav/thunder-client-support#path

**Reply by AmirHo3einJS** [reply]
thank you, I don't know why I didn't see that 🤣
